### ISSUE TITLE
Python string unpacking broken again

### ISSUE THREAD

**Issue by rosstitmarsh** [eclipse/tahu#366]
If the array has any any 0 length strings at the start or the end, the strip function on line 129 in convert_from_packed_string_array removes them.

e.g.
```python
a = ["", "foo", "", "", "bar", "", ""]
convert_from_packed_string_array(convert_to_packed_string_array(a))
# returns ["foo", "", "", "bar"]
```

Replacing `strip` with `removesuffix` would work in python >=3.9 or removing the `strip` and putting the `[:-1]` back on the end.

**Comment by Alain-Godo** [comment]
Hi @rosstitmarsh, recently I did a PR that can solve that issue ( and others ). I'm waiting for the review-> https://github.com/eclipse/tahu/pull/367

**Comment by Alain-Godo** [comment]
![image](https://github.com/eclipse/tahu/assets/125923673/8883838c-e2ed-4007-a3c2-344ded905cdf)

**Comment by rosstitmarsh** [comment]
Fixed in #367
